test/unit_tests/models/test_modules.py

```diff
@@ -94,6 +93,9 @@ def test_dense_spatial_filter_forward_collapse_false():
     assert output.shape[:2] == torch.Size([5, 33])
 
 
+# Issue with False, False option
+@pytest.mark.skipif(platform.system() == 'Linux',
+                    reason="Not supported on Linux")
```

Comment:
why?

Reply:
We don't know to be honest, but we opened issue #583